### ISSUE TITLE
Watermark cannot be removed

### ISSUE THREAD

**Issue by SmileSnail5470** [Aitchson-Hwang/MNet#2]
Using a custom 256×256 image with a visible watermark, the MNet model is unable to remove the watermark.
input:

![Image](https://github.com/user-attachments/assets/cbce0f54-64ee-4f66-8c49-194596e8e595)

output:

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/5431ed91-683a-4d13-a923-3e705db66287" />

output mask:

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/99c766a1-7e3b-4376-8ef3-1a72b6bfe3a4" />

主要代码如下

`
imoutput, immask = model(tile)
imfinal = imoutput * immask + tile * (1 - immask)

def _load_model(self, resume_path):
        args = {
            "k1": 5,
            "k2": 2,
            "k3": 1,
            "srm": 1,
            "inf": 1,
            "crf": 1,
            "sharing": 1,
        }
        model = MNetold(wf=54, scale=42, vscale=42, args=args)
        current_checkpoint = torch.load(resume_path, map_location="cpu", weights_only=False)
        model.load_state_dict(current_checkpoint['state_dict'], strict=True)
        model.eval()
        return model

`
